Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -197,12 +198,11 @@ def run_checks(
                                                          .select { |items| items.length > 1 }
                                                          .flatten
       end
+      formulae_and_casks_total = formulae_and_casks_to_check.count
```

Comment:
Nit: This is only used in one place so it should be defined near where it's used or removed and just called directly in the if condition.